src/pages/cartPage/CartPage.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useData } from "../../context/data-context";
+import "./cartPage.css";
+import { removeFromCart, updateQuantity } from "../../utility";
+import { PriceSummary } from "../../component";
+
+const CartPage = () => {
+  const { state, dispatch } = useData();
+
+  return (
+    <>
+      {state.cart && state.cart.length > 0 ? (
+        <section className="cart-section">
+          <div className="cart-container flex">
+            <div className="cart-item-container card-horizontal-style">
+              <h2 className="cart-heading">
+                My Cart (<span>{state.cart.length}</span>)
+              </h2>
+              {/* card */}
+              {state.cart.map((product) => {
+                const {
+                  _id,
+                  brand,
+                  title,
+                  priceAfterDiscount,
+                  discount,
+                  Image,
+                  alt,
+                } = product;
+                return (
+                  <div className="card card-Horizontal" key={_id}>
+                    <div>
+                      <img className="image-responsive" src={Image} alt={alt} />
+                    </div>
+                    <div className="card-container">
+                      <div className="card-text-container">
+                        <p>{brand}</p>
+                        <h5 className="card-heading">{title}</h5>
+                        <p className="card-subheading">
+                          ₹ {priceAfterDiscount}
+                        </p>
+                        <span className="small-text">{discount}% off</span>
+                        <div className="quantity flex flex-align-center">
+                          <span>Quantity : </span>
+                          <div className="counter">
+                            <span className="counter-item">
+                              <i
+                                className="fas fa-minus"
+                                onClick={() =>
+                                  updateQuantity(_id, dispatch, "decrement")
+                                }
```

Comment:
the qty goes below negative, either disable the btn or hide the btn

Reply:
Thanks for pointing out will fix that